.auto-claude/specs/020-task-appears-stuck-task-runner-crash-loop-file-not/init.sh

```
@@ -0,0 +1,75 @@
+YELLOW='\033[1;33m'
+NC='\033[0m'
+
+# Wait for service function
+wait_for_service() {
+    local port=$1
+    local name=$2
+    local max=30
+    local count=0
+
+    echo "Waiting for $name on port $port..."
+    while ! nc -z localhost $port 2>/dev/null; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)